jakarta-eclipse/org.eclipse.lsp4jakarta.tests/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/jsonb/ExtraJsonbCreatorAnnotations.java

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.sample.jakarta.jsonb;
+
+import jakarta.json.bind.annotation.JsonbCreator;
+
+public class ExtraJsonbCreatorAnnotation {
```

Comment:
class name is missing `s`:  ExtraJsonbCreatorAnnotations, class can be renamed to singular or class name and associated constructors should be updated

Reply:
371ec55